tests/source/let_else.rs

```diff
@@ -160,3 +160,23 @@ fn with_trailing_try_operator() {
     // Maybe this is a workaround?
     let Ok(Some(next_bucket)) = ranking_rules[cur_ranking_rule_index].next_bucket(ctx, logger, &ranking_rule_universes[cur_ranking_rule_index]) else { return };
 }
+
+fn issue5901() {
+    #[cfg(target_os = "linux")]
+    let Some(x) = foo else { return; };
```

Comment:
I appreciate you adding both v1 and v2 test cases for this.

Is there a reason why these test cases use `return;` instead of `todo!()` like the last one?

Because `return;` is a statement rustfmt will always rewrite the else block over multiple lines in those cases.


Also, I'd like to increase the coverage with attributes. When you get a chance please take some of the longer test cases from the v1 test file and add attributes to them and incorporate them in the v2 test file (the more the better 😁).

Reply:
> Is there a reason why these test cases use return; instead of todo!() like the last one?

Actually there is no reason. I'll replace `return`s with `todo!`.

> When you get a chance please take some of the longer test cases from the v1 test file and add attributes to them and incorporate them in the v2 test file

Sure. I'll try to do that.